test/blackbox-tests/test-cases/pkg/dev-tool-conflict-test.t

```diff
@@ -0,0 +1,256 @@
+Testing the conflicts that could occur between the dependencies of `dune-project`
+and dev-tool dependencies.
```

Comment:
Please check that my changes didn't affect the accuracy of this sentence. I fixed a verb tense issue and other grammar, but I want to ensure I didn't change the meaning.

Reply:
Thanks, that is OK.